aurora_module/variables.tf

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2011 - 2017, Coveo Solutions Inc.
+ */
+
+// Required variables :
+
+variable "master_password" {
+  description = "Password for the master DB user"
+}
+
+variable "subnet_ids" {
+  type        = "list"
+  description = "A list of VPC subnets IDs"
+}
+
+variable "vpc_id" {
+  type = "string"
+  description = "VPC ID"
+}
+
+// Optional variables :
+
+variable "instances_count" {
+  description = "Number of instances"
+  default = 1
+}
+
+
+variable "custom_identifier" {
+  description = "An identifier to be used as a default value to identified your ressources."
+}
+
+variable "rds_cluster_db_param_group_params" {
+  type = "list"
+  default = []
+  description = "A list of cluster DB group params."
+}
+
+variable "rds_db_param_group_params" {
+  type = "list"
+  default = []
+  description = "A list of DB group params."
+}
+
+variable "optional_parameters" {
+  type        = "map"
+  description = "Parameters with default values. Map key corresponds to rds_db_cluster and rds_db_cluster_instance attributes; if no key is present default value is used."
+  default     = {}
+}
+
+variable "encryption_kms_key_id" {
```

Comment:
This is unused? It should be used. We should allow this module users to provide their own KMS key 🙂

Reply:
Hmm, I must have forgotten to use it.